### ISSUE TITLE
Support create FloatingActionsMenu programmatically

### ISSUE THREAD

**Issue by darktiny** [futuresimple/android-floating-action-button#172]
* Attributes can't be changed programmatically
* Dynamically addButton to FloatingActionsMenu, `mButtonsCount` doesn't set correctly.

for example:

````
FloatingActionsMenu menu = new FloatingActionsMenu(getContext());
menu.addButton(new FloatingActionButton(getContext()));
menu.addButton(new FloatingActionButton(getContext()));
menu.addButton(new FloatingActionButton(getContext()));
LayoutParams params = new LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT, Gravity.RIGHT | Gravity.BOTTOM);
addView(menu, params);
````

It support to be 3 buttons when expanded, however, only 2 buttons.

It seems `onFinishInflate()` not be called when creating FloatingActionsMenu programmatically. Set `mButtonsCount`'s initial value to 1 fix the bug though, but I'm not sure whether it's the best solution.

**Comment by Andrew-Quebe** [comment]
+1

**Comment by TakumaMochizuki** [comment]
+1, and please support change FloatingActionsMenu background color programmatically. :)

**Comment by chalup** [comment]
Incorrect `mButtonsCount` value was fixed in #228, the request for FAM setters is already tracked at #49
